addons/resource/tests/test_resource_calendar.py

```diff
@@ -65,3 +67,29 @@ def test_flexible_calendar_attendance_interval_duration(self):
         actual_duration = end - start
 
         self.assertEqual(actual_duration.seconds / 3600, calendar.full_time_required_hours, "For a full day, the interval must match full time required hours")
+
+    @freeze_time("2019-5-28 08:00:00")
+    def test_working_time_holiday_multicompany(self):
+        """
+        This test checks that there is no issue computing the "working time to assign" even if a holiday has been set
+        for this moment, but on another company.
+        """
+        company_0, company_1 = self.env['res.company'].create([{
+            "name": "Test company 0"},
+            {
+                "name": "Test company 1"}])
```

Comment:
Nitpicking for better readability :innocent: 
```suggestion
        company_0, company_1 = self.env['res.company'].create([{
            "name": "Test company 0",
        },
        {
            "name": "Test company 1",
        }])
```